### ISSUE TITLE
Diagrams and variation captions don't need a period before the number anymore

### ISSUE THREAD

**Issue by dormerod** [Kashomon/gpub#19]
Presently the beginning of the caption for diagrams and variations is generated as "Diagram. x ..." or "Variation. x ..." respectively.

I think the period after Diagram/Variation was required before because it was used to denote the abbreviation of Dia. x or Var. x. However, now that these labels have been expanded to the full word the period is redundant and I think it would be better to remove it.

In short, simply "Diagram x ..." and Variation x ..." seems to be better.

(Where other books use a period in between it's because they're abbreviating the word.)

**Comment by Kashomon** [comment]
easy fix. I'll push it out tomorrow

**Comment by dormerod** [comment]
Fixed in 6fa84ae
